src/main/java/gil/server/Response.java

```diff
@@ -32,8 +30,53 @@ public String getContentLength() {
         return contentLength;
     }
 
-    public String getBody() {
-        return body;
+    public String getDate() {
+        return date;
+    }
+
+    public String getProtocol() {
+        return protocol;
     }
 
+    public String getReasonPhrase() {
+        return reasonPhrase;
+    }
+
+    public String getStartLine() {
+        return protocol + " " + statusCode + " " + reasonPhrase;
```

Comment:
Assign strings in this file to either variables or constants, whichever seems more relevant.